vocabulary-draft/README.md

```diff
@@ -0,0 +1,31 @@
+# Continuous Delivery Events
+
+Continuous Delivery Events (CD Events or CDE) is a common event format for the Continuous Delivery (CD) ecosystem.
```

Comment:
Do we want to have something else here like.
> We see these events as an enabler for interoperability between different CI/CD tools as the communication is tool agnostic

Reply:
Thanks @m-linner-ericsson - I will add it.